apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -69,14 +69,14 @@ const getValidParamValues = (filterKey, paramValues) => {
 };
 
 // Returns initial filter states based on URL parameters (returns empty filters if
-// no relevant parameters in the URL). The filter params are of the form:
-// "filter_name:checked_value_1,checked_value_2,etc."
+// no relevant parameters in the URL).
 const getInitialFilterStates = () => {
   const filterTypeKeys = Object.keys(filterTypes);
   const urlParams = queryParams();
 
   let filters = getEmptyFilters();
   Object.keys(urlParams).forEach(paramKey => {
+    // Ensure valid filter key
```

Comment:
Nit: I think this comment and the ones below it in the `handleUpdateFilter` are self-documenting in the code and aren't necessary